src/Umbraco.Core/Security/UmbracoBackOfficeIdentity.cs

```diff
@@ -1,233 +0,0 @@
-            if (HasClaim(x => x.Type == Constants.Security.AllowedApplicationsClaimType) == false && allowedApps != null)
-            {
-                foreach (var application in allowedApps)
-                {
-                    AddClaim(new Claim(Constants.Security.AllowedApplicationsClaimType, application, ClaimValueTypes.String, Issuer, Issuer, this));
-                }
-            }
-
-            //Claims are added by the ClaimsIdentityFactory because our UserStore supports roles, however this identity might
-            // not be made with that factory if it was created with a different ticket so perform the check
-            if (HasClaim(x => x.Type == DefaultRoleClaimType) == false && roles != null)
-            {
-                //manually add them
-                foreach (var roleName in roles)
-                {
-                    AddClaim(new Claim(RoleClaimType, roleName, ClaimValueTypes.String, Issuer, Issuer, this));
-                }
-            }
-
-        }
-
-        /// <inheritdoc />
-        /// <summary>
-        /// Gets the type of authenticated identity.
-        /// </summary>
-        /// <returns>
-        /// The type of authenticated identity. This property always returns "UmbracoBackOffice".
-        /// </returns>
-        public override string AuthenticationType => Issuer;
-
-        private int[] _startContentNodes;
-        public int[] StartContentNodes => _startContentNodes ?? (_startContentNodes = FindAll(x => x.Type == Constants.Security.StartContentNodeIdClaimType).Select(app => int.TryParse(app.Value, out var i) ? i : default).Where(x => x != default).ToArray());
-
-        private int[] _startMediaNodes;
-        public int[] StartMediaNodes => _startMediaNodes ?? (_startMediaNodes = FindAll(x => x.Type == Constants.Security.StartMediaNodeIdClaimType).Select(app => int.TryParse(app.Value, out var i) ? i : default).Where(x => x != default).ToArray());
-
-        private string[] _allowedApplications;
-        public string[] AllowedApplications => _allowedApplications ?? (_allowedApplications = FindAll(x => x.Type == Constants.Security.AllowedApplicationsClaimType).Select(app => app.Value).ToArray());
-
-        public int Id => int.Parse(this.FindFirstValue(ClaimTypes.NameIdentifier));
-
-        public string RealName => this.FindFirstValue(ClaimTypes.GivenName);
-
-        public string Username => this.FindFirstValue(ClaimTypes.Name);
-
-        public string Culture => this.FindFirstValue(ClaimTypes.Locality);
-
-        public string SecurityStamp => this.FindFirstValue(Constants.Security.SecurityStampClaimType);
-
-        public string[] Roles => FindAll(x => x.Type == DefaultRoleClaimType).Select(role => role.Value).ToArray();
-
-        /// <summary>
-        /// Overridden to remove any temporary claims that shouldn't be copied
-        /// </summary>
-        /// <returns></returns>
-        public override ClaimsIdentity Clone()
```

Comment:
Seems wrong we are not handling this anymore.

@Shazwazza, do you have any suggestions regarding the `Clone` method. It is called by many of Microsoft's internal methods. IMO it seems dangerous to save `Constants.Security.TicketExpiresClaimType` as a claim now that we can't override the clone method?

Currently, if I re-introduce the clone test, it shows this is not handled anymore.